### PR TITLE
Add PMJ-based CSA in `batch_processing.sh`

### LINKED ISSUE #3495: Add PMJ-based CSA method in `batch_processing.sh`
## Description 
Now that the PMJ-based method used to compute CSA in `sct_process_segmentation` is completed (https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3478), we need to add an example of the usage in `batch_processing.sh` and the related tests.

### PR DESCRIPTION
**Pull request by sandrinebedard**
#### GitHub

- [x] I've given this PR a concise, self-descriptive, and meaningful title
- [x] I've linked relevant issues in the PR body
- [x] I've applied [the relevant labels](https://www.neuro.polymtl.ca/software/contributing#pr_labels) to this PR
- [x] I've applied a [release milestone](https://github.com/spinalcordtoolbox/spinalcordtoolbox/milestones) (major, minor, patch) in line with [Semantic Versioning guidelines](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Misc%3A-Creating-a-new-release#convention-for-naming-releases) 
- [x] I've assigned a reviewer

#### PR contents

- [x] I've consulted [SCT's internal developer documentation](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki) to ensure my contribution is in line with any relevant design decisions
- [x] I've added [relevant tests](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Programming%3A-Tests) for my contribution
- [ ] I've updated the [relevant documentation](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Programming%3A-Documentation) for my changes, including argparse descriptions, docstrings, and ReadTheDocs tutorial pages

## Description
This PR addresses #3495 . Since PMJ-based CSA method is now merged (#3478), this PR adds the usage of this method in `batch_processing.sh` script and adds the related tests in `testing/batch_processing/test_batch_processing.py`.

## Linked issues
Resolves #3495

### CODE REVIEW

File: testing/batch_processing/test_batch_processing.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 # TODO: We can and should be verifying more results produced by this pipeline, but which values?
 TESTED_VALUES = [("t2/csa_c2c3.csv", 0, "MEAN(area)"),
+                 ("t2/csa_pmj.csv", 0, "MEAN(area)"),
```
Comment (joshuacwnewton) [inline testing/batch_processing/test_batch_processing.py:19]: The current test checks the value against [cached result files](https://github.com/spinalcordtoolbox/spinalcordtoolbox/tree/master/testing/batch_processing/cached_results), so we would need to also commit the file `t2/csa_pmj.csv` generated by `batch_processing.sh` in order for the test to pass.

**Review by joshuacwnewton — CHANGES_REQUESTED** [review]
Thank you so much for taking the initiative on this! :heart:

File: testing/batch_processing/test_batch_processing.py [hunk 2]
```diff
@@ -16,6 +16,7 @@
 
 # TODO: We can and should be verifying more results produced by this pipeline, but which values?
 TESTED_VALUES = [("t2/csa_c2c3.csv", 0, "MEAN(area)"),
```
Comment (joshuacwnewton) [inline testing/batch_processing/test_batch_processing.py:18]: One other neat thing: By adding the CSV file to the "`TESTED_VALUES`" list, your CSA (pmj) value now automatically shows up [in output of the batch_processing.sh script](https://github.com/spinalcordtoolbox/spinalcordtoolbox/runs/3231834684#step:4:3558), too! :smile:

File: testing/batch_processing/test_batch_processing.py (hunk 1 above)
Comment (sandrinebedard) [reply testing/batch_processing/test_batch_processing.py:19]: oups,  forgot to push the commit! done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3497/commits/5e77268f6d973c00ea1323928e56b92a16948305

**Review by joshuacwnewton — APPROVED** [review]
Thanks for keeping your PR small and easy to review! :tada:

**Pull request merged**